scripts/reana

```diff
@@ -135,6 +222,7 @@ usage () {
     echo "  install-minikube    Installs and starts local minikube instance"
     echo "  init                Initialize REANA system"
     echo "  down                Delete REANA components"
+    echo "  demo helloworld     Runs a helloworld demo"
```

Comment:
`run helloworld` so that we are ready for the future `run myanalysis` without having to change the CLI.